packaging/flatpak/org.openwinecomponents.umu.launcher.yml

```diff
@@ -143,14 +148,14 @@ modules:
   - name: umu-run
     buildsystem: simple
     build-commands:
-      - install -D umu-run-cli /app/bin/umu-run
-      - install -D umu-launcher.tar.gz /app/share/umu/umu-launcher.tar.gz
+      - |
+        git submodule update --init --recursive
+        ./configure --prefix=/app
+        make install
     sources:
-      - type: file
-        path: umu-run-cli
-      - type: file
-        url: https://github.com/Open-Wine-Components/umu-launcher/releases/download/0.1-RC3/umu-launcher.tar.gz
-        sha256: e25c4dd0636d04e7c8c534cf3c5bbdca5ae0d49f146ee8395306174700899952
+      - type: git
+        url: https://github.com/Open-Wine-Components/umu-launcher.git
+        branch: main
```

Comment:
This won't build since the changes are not merged in main, so will probably have to create another branch